Makefile

```diff
@@ -552,12 +552,12 @@ build-capi-headless-ios:
 
 # test compilers
 test-stage-0-wast:
-	$(CARGO_BINARY) nextest run $(CARGO_TARGET_FLAG) --release $(compiler_features) --locked
+	$(CARGO_BINARY) nextest run $(CARGO_TARGET_FLAG) --release $(compiler_features) --locked --jobs=1
```

Comment:
Why 1 job?

Reply:
In CI it already uses 1 job. While not in CI, it can happen that two different tests write on/read the same file, and using a single job mitigates this problem.